src/gf-oemid

```diff
@@ -23,14 +23,19 @@ cp --reflink=auto ${src} ${tmp_dest}
 chmod u+w ${tmp_dest}
 
 coreos_gf_run_mount "${tmp_dest}"
-# For now we just modify the grub config, but *not* the /boot/loader
-# entry that generated it, because...well it's simpler and in theory
-# we only need the OEM ID for the first boot where Ignition runs.  Might
-# as well not have other things using it persistently.
-grubcfg_src=/boot/grub2/grub.cfg
-coreos_gf download ${grubcfg_src} ${tmpd}/grub.cfg
+
+# Inject OEM label in all relevant places:
+# * grub config
+# * BLS config (for subsequent config regeneration)
+grubcfg_path=/boot/loader/grub.cfg
+coreos_gf download ${grubcfg_path} ${tmpd}/grub.cfg
 sed -i -e 's,^\(linux16 .*\),\1 coreos.oem.id='${oemid}',' ${tmpd}/grub.cfg
-coreos_gf upload ${tmpd}/grub.cfg ${grubcfg_src}
+coreos_gf upload ${tmpd}/grub.cfg ${grubcfg_path}
+blscfg_path=/boot/loader/entries/ostree-1-coreos.conf
```

Comment:
The `coreos` identifier here is arbitrary - it might be changed in a custom build.  Further the use of `1` here is an ostree implementation detail that [did change recently](https://github.com/ostreedev/ostree/pull/1654).

Maybe best would be to use the guestfish `glob` verb like `blscfg_path=$(coreos_gf glob ls /boot/loader/entries/*.conf)` ?

Reply:
Yup, it sounds better indeed. Is the star-wildcard somehow limitable? That is:
 * is the `ostree-` prefix static? (I think so)
 * is this repo the primary source of the os identifier? (I think it is the stateroot and I think it's defined [here](https://github.com/coreos/coreos-assembler/blob/59d51013d6f906a38cc0af610328574d3a630f53/src/cmd-build#L139), but I don't know if it comes from somewhere else too)

Reply:
You are correct on both counts. Though I could imagine that `coreos` string become configurable, so I'd just stick with `ostree-*.conf` I think. At the time it's running, there really should be only one entry in there.